src/analytics/analytics-step-tracker.ts

```diff
@@ -99,7 +99,7 @@ export default class AnalyticsStepTracker implements StepTracker {
         this._checkoutStarted = true;
     }
 
-    trackOrderComplete(): void {
+    trackOrderComplete(features?: Record<string, boolean>): void {
```

Comment:
might be worth adding a docblock explaining how `features work`.

The docblock will automatically be included in the generated documentation